dev/js/components/CustomDropdown/style.js

```diff
@@ -0,0 +1,137 @@
+import { css } from '@emotion/css';
+import { mobileScreenWidth } from '@constants';
+import { primaryFont } from '@constants'
+
+export const ddContainer = css`
+    background: #FCFCFC;
+    box-shadow: 0px 6.74px 16.85px rgba(191, 21, 108, 0.05);
+    border-radius: 6.74px;
+    width: 230px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top:20px;
+
+
+    @media(max-width: ${mobileScreenWidth}){
+        width: 100%;
+        font-size:14px;
+        height: 32px;
+    };
+`
+export const ddContainerItem = css`
+    background: #FCFCFC;
+    border: none;
+    width: 100%;
+    border-top:0.8px solid #EAEAEF;
+    font-family: ${primaryFont};
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 27px;
+    cursor:pointer;
+    color: #363B64;
+    text-align:left;
+    margin-right:0px;
+    :hover{
+       
+        background: white;
+        outline:none;
+        background-color:#EAEAEF
+    }
+    :focus{
+        
+        background: none;
+        outline:none;
+        background-color:#EAEAEF
+    }
+    :active{
+        
+        background: none;
+        outline:none;
+        background-color:#EAEAEF
+    }
+`
+
+export const ddContainerItemHeader = css`
+background: #FCFCFC;
+border: none;
+width: 100%;
+font-family: ${primaryFont};
+font-style: normal;
+font-weight: normal;
+font-size: 18px;
+line-height: 27px;
+cursor:pointer;
+color: #A098AE;
+text-align:left;
+background-image:url("./../../../assets/images/common/chevron-down.svg");
```

Comment:
Please use path name using "@" alias that we use or using CONST {ASSET_BASE_URL}